src/test/kotlin/org/rust/ide/navigation/goto/RustGotoSuperHandlerTest.kt

```diff
@@ -0,0 +1,20 @@
+package org.rust.ide.navigation.goto
+
+import com.intellij.lang.CodeInsightActions
+import org.rust.lang.RustLanguage
+import org.rust.lang.RustTestCaseBase
+
+class RustGotoSuperHandlerTest : RustTestCaseBase() {
+    override val dataPath = "org/rust/ide/navigation/fixtures/goto_super"
+
+    fun testGotoSuper() {
+        myFixture.copyFileToProject("main.rs", "/main.rs")
```

Comment:
sorry, i just leave the `main.rs` here intentional, cause i did not fully understand this https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/lang/RustTestCaseBase.kt#L85, i can not find any code that copy the `main.rs` file to the test project directory. so how can it resolve bin target when processing test fixtures?